### PR TITLE
Created login and signup pages

### PR DESCRIPTION
**Pull request by crow-418**
Created login and signup boxes

Added html for login form

Added .css file for styling the registration forms

Added html to signup page and added the required css properties from registration.css

Applied standard naming conventions to the files

### CODE REVIEW

File: html/crypto.html [hunk 1]
```diff
@@ -17,8 +17,8 @@
                 <li><a href="purchases.html">Purchases</a></li>
             </ul> 
             <ul class="nav">
-                <li><a href="#">Log In</a></li>
-                <li><a href="#">Sign Up</a></li>
```
Comment (Filip-Nachov) [inline html/crypto.html:21]: No need to delete this just bringing up the removal count

File: html/index.html [hunk 2]
```diff
@@ -17,8 +17,8 @@
                 <li><a href="purchases.html">Purchases</a></li>
             </ul> 
             <ul class="nav">
-                <li><a href="sign-in.html">Sign In</a></li>
-                <li><a href="sign-up.html">Sign Up</a></li>
```
Comment (Filip-Nachov) [inline html/index.html]: No need to delete this

Comment (Filip-Nachov) [reply html/index.html]: like how you changed sign in into Log in

Comment (crow-418) [reply html/index.html]: that needs to be deleted. Because there's no more sign-in.html there.s a login.html though

**Review by Filip-Nachov — APPROVED** [review]
LGTM! like how you only have 7 lines removed and 222 added

**Comment by Filip-Nachov** [comment]
No ok i commented at a other place accidentally that was a mistake from me

________________________________

**Pull request merged**